pwa/src/components/applications/applicationsTable.tsx

```diff
@@ -59,17 +62,19 @@ export default function ApplicationsTable() {
   };
 
   const handleDeleteApplication = (id): void => {
-    if (confirm(`Do you want to delete this application?`)) {
-      API.Application.delete(id)
-        .then(() => {
-          setAlert({ message: `Deleted application`, type: "success" });
-          handleSetApplications();
-        })
-        .catch((err) => {
-          setAlert({ title: "Oops something went wrong", message: err, type: "danger" });
-          throw new Error("DELETE application error: " + err);
-        });
-    }
+    setLoadingOverlay(true);
+    API.Application.delete(id)
+      .then(() => {
+        setAlert({ message: `Deleted application`, type: "success" });
+        handleSetApplications();
+      })
+      .catch((err) => {
+        setAlert({ message: err, type: "danger" });
```

Comment:
You are (unintentionally I assume) overwriting some changes made to this `setAlert()`. The previous one had a title and a message instead of only the message.

Reply:
Check this throughout your updated files please.

Reply:
Check this throughout your updated files please.